code401Challenges/src/test/java/code401Challenges/hashtable/TreeIntersectionTest.java

```diff
@@ -1,7 +1,114 @@
 package code401Challenges.hashtable;
 
+import code401Challenges.tree.Node;
+import org.junit.Test;
 import static org.junit.Assert.*;
 
+import java.util.HashSet;
+
 public class TreeIntersectionTest {
 
-}
\ No newline at end of file
+    @Test
+    public void treeIntersectionTest1 () {
+        //              10
+        //             /  \
+        //            5    15
+        //           / \   / \
+        //          2  7  12  17
+
+
+        Node root1 = new Node(10,
+                new Node(5, new Node(2), new Node(7)),
+                new Node(15, new Node(12), new Node(17)));
```

Comment:
Your Node class uses generics, so use the diamond operator to declare what type of Node it is, like in this case : `Node<Integer> root1 = new Node<>(10, . . . `